org.mozilla.Thunderbird.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id": "org.mozilla.Thunderbird",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "thunderbird",
+    "rename-icon": "thunderbird",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* OpenGL access */
+        "--device=dri",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Access to thunderbird configuration and cache files */
+        "--filesystem=~/.cache/thunderbird", "--filesystem=~/.thunderbird",
+        /* Access to home directory (for attachments) */
+        "--filesystem=home",
+        /* Needed for dconf to work */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "modules": [
+        {
+            "name": "flatpak-xdg-open-shim",
```

Comment:
This can be removed an xdg-open shim in the runtime.

Reply:
Thanks, this was somebody elses project so I wasn't sure.

Reply:
@TingPing Correct. IIRC this was added at a time where upstream's xdg-utils wouldn't work, but since the freedesktop runtime already includes [flatpak-xdg-utils](https://github.com/flatpak/flatpak-xdg-utils), this shouldn't be necessary now